pkg/apis/migration/v1alpha1/gvk.go

```diff
@@ -0,0 +1,29 @@
+package v1alpha1
+
+// UnsupportedNamespace - namespace, which is noticed
+// to contain resources unsupported by the migration
+type UnsupportedNamespace struct {
+	Name                 string   `json:"name"`
+	UnsupportedResources []string `json:"unsupportedResources"`
```

Comment:
This should just be `Resources` within the scope of `UnsupportedNamespaces`.

Reply:
This will complicate understanding for this field in the status. Either the name, or some sub item should tell the reason for it being there.